public/app/core/logs_model.ts

```diff
@@ -106,7 +107,8 @@ export function makeSeriesForLogs(rows: LogRowModel[], intervalMs: number): Grap
   const bucketSize = intervalMs * 10;
   const seriesList: any[] = [];
 
-  for (const row of rows) {
+  const sortedRows = rows.sort(sortInAscendingOrder);
```

Comment:
@hugohaggmark I have updated this and used the `sortInAscendingOrder` from explore utils, that uses timestamp for sorting. Great point! 🙂